packages/terra-button/src/_mixins.scss

```diff
@@ -75,3 +75,28 @@
     color: map-get($color-map, disabled-color);
   }
 }
+
+@mixin terra-consumer-theme-button {
+  --terra-button-background-color-primary: var(--terra-consumer-color);
```

Comment:
If we're using SASS, just pass [keyword arguments](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#Keyword_Arguments0). This will result in a more terse theme description so we don't have variables  assigned in the cascade ahead of variable mappings.